flower_reader.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+FILENAME = 'flowers.txt'
```

Comment:
@paulghaddad I need to break down this file into a main file and an exercise file. But before I'd like this first iteration to just look at the code and see if I need to refactor further. 

Also, thoughts on the constants? Pylint suggested I change them to constants.